### ISSUE TITLE
Typescript definitions not usable

### ISSUE THREAD

**Issue by MicroTransactionsMatterToo** [Codesleuth/esendex-node#24]
While the typescript definitions are there, they're not in any location that typescript will look for them. As well as this, the typings are split in to separate files, when they should be contained in a single `index.d.ts` file. I would greatly appreciate it if this was remedied, as currently the module isn't usable in other typescript projects without tweaking.

**Comment by Codesleuth** [comment]
This is a Typescript project, written in Typescript. I can see the build is generating definitions output, but it really shouldn't.

The problem here is that this was written back when the definitions worked perfectly well, and now the preference is to just have the original source be the definitions. It's not too difficult to remedy.

As always, I welcome pull requests. However, there is no release schedule cadence agreed with Esendex, so that might be an issue.

**Comment by MicroTransactionsMatterToo** [comment]
Yeah, I realised it was written in typescript. I mostly just wanted to get it sorted out so that in future we can update the library without issue. Otherwise you often forget about the tweaks you had to make to get it to work. How would I go about getting typescript to use the correct files, preferably without modifying your library. I wasn't able to get it to work by tweaking the tsconfig

**Comment by alisalahio** [comment]
I agree that project won't be usable for most developers in Typescript projects

But as a fast remedy, you may use replace
`import Esendex from 'esendex'` with
`const Esendex = require('esendex')`

Which will work fine
